mods/ctf/ctf_map/ctf_map_core/schem_map.lua

```diff
@@ -230,6 +230,11 @@ local function shuffle_maps()
 		local j = math.random(i)
 		shuffled_order[i], shuffled_order[j] = shuffled_order[j], shuffled_order[i]
 	end
+	if shuffled_order[1] == not_first then
```

Comment:
A comment above this line explaining what this check is and what this block does, would be very useful.